src/pages/Login/Main/Main.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { signInWithEmailAndPassword } from 'firebase/auth';
+import { auth } from '../../../config/firebase';
+import { useNavigate } from 'react-router-dom';
+import "./Main.css"
+
+
+
+const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const navigate = useNavigate();
+
+  const handleLogin = async (event) => {
+    event.preventDefault();
+    try {
+      await signInWithEmailAndPassword(auth, email, password);
+      console.log("Logged in!");
+      navigate('/menu');
```

Comment:
You don't store user status anywhere it will prevent you to know the status after this component will be unmounted